device/iot-device-samples/file-upload-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/FileUploadSample.java

```diff
@@ -142,11 +146,14 @@ private static void uploadFile(DeviceClient client, String baseDirectory, String
 
             try
             {
-                // Note that other versions of the Azure Storage SDK can be used here instead. The latest can be found here:
-                // https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage#azure-storage-sdk-client-library-for-java
```

Comment:
No need for this note anymore since we are using the latest version. We only added this note previously to encourage users to avoid this older storage SDK version

Reply:
Isn't this a good information to relay anyway, that they are free to use any version of the storage sdk with our clients?